index.js

```diff
@@ -2,11 +2,12 @@
 	MIT License http://www.opensource.org/licenses/mit-license.php
 	Author Tobias Koppers @sokra
 */
+const jsesc = require('jsesc');
+
 module.exports = function(content) {
 	this.cacheable && this.cacheable();
 	this.value = content;
-	content = JSON.stringify(content)
-		.replace(/\u2028/g, '\\u2028')
-		.replace(/\u2029/g, '\\u2029');
-	return `module.exports = ${content}`;
+	content = JSON.stringify(content);
+	content = jsesc(content);
```

Comment:
You wouldn’t use both. You’d use `jsesc(content, { wrap: true ))` instead of `JSON.stringify(content)` in this case.

Reply:
Would you mind reviewing [`json-loader` #62](https://github.com/webpack-contrib/json-loader/pull/62) aswell please ?